Toolbox/final/cleanup.sh

```diff
@@ -1,17 +1,21 @@
 #!/bin/ksh
 # This script will cleanup old MIB Toolbox installations. 
 
-mount -uw /mnt/app
+echo "Cleanup, mounting app rw"
+on -f mmx mount -uw /mnt/app
 
 echo "Deleting old mqbcoding.esd pre v4.1"
-rm /mnt/app/eso/hmi/engdefs/mqbcoding.esd
-rm -r /mnt/app/eso/hmi/engdefs/mqbcoding.esd.*
+echo "___________________________________" 
+# Pass the command as a string to remote ksh preserve the *
+on -f mmx ksh -c 'rm -rvf /eso/hmi/engdefs/mqbcoding.esd*'
```

Comment:
Will this also delete the mqbcoding.esd (see old line 7) because old stuff are 3 files:
mqbcoding.esd
mqbcoding.esd.fileinfo (or some sort of)
mqbcoding.esd.checksum (or some sort of)
All three should be delted.

Reply:
Yep, * also matches zero characters, so yep it would get rid of the original/base file.

Reply:
Okay. I thought a * indicator at the end awaits more characters.

Reply:
I'm pretty certain this will be fine... But to be honest my mqbcoding.esd had been deleted by a previous trial so it hasn't had a proper end-to-end test